.github/pull_request_template.md

```diff
@@ -32,9 +32,9 @@ We're here to help! This is simply a reminder of what we are going to look for b
 
 - [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
 - [ ] I have added tests that prove my fix is effective or that my feature works
-- [ ] I have added any necessary documentation (if appropriate)
+- [ ] I have added any necessary documentation listed in document requirements listed in [CONTRIBUTING.md](https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md) (if appropriate)
```

Comment:
Suggest to change as follows: 

[ ] If I had to update any .go files, I have added missing doc annotations listed in [CONTRIBUTING.md], (https://github.com/mongodb/mongocli/blob/master/CONTRIBUTING.md), or reviewed existing annotations to check that each command has a description, an example, and all of command's arguments have descriptions.

Reply:
This seems a bit too long and the intention of adding the details into CONTRIBUTING is so that we don't need to specify any "more important" things that need to be done. I think if we just mention "followed document " in the file it should be fine? What do you think?